credsweeper/secret/config.json

```diff
@@ -129,6 +129,8 @@
         ".jks"
     ],
     "check_for_literals": true,
+    "min_pattern_value_length": 12,
+    "min_keyword_value_length": 4,
```

Comment:
4 symbols in base64 may encode 3 normal:
```
$ echo -n '{"a' | base64
eyJh
```
probably 6 is enough 
```
$ echo -n '{"a":' | base64
eyJhIjo=
```

Reply:
> 4 symbols in base64 may encode 3 normal:

@babenek 
I think question should be if 4/6 symbol json can realistically be a credential
'{"a":' is not even a valid json...

`echo -n '{0:0}' | base64` -> len 8
`echo -n '{"a":0}' | base64` -> len 12

So minimal length of 12 (`min_pattern_value_length`) would require payload to be at least not smaller than `{"a":0}`

Reply:
@babenek 
Added negative examples with 35153123d922d557743fe91687d030c6575ff9b3